### PR TITLE
Fix macros expecting a range as option

### PR DESCRIPTION
**Pull request by waiting-for-dev**
References #608

### CODE REVIEW

File: lib/dry/validation/rule.rb [hunk 1]
```diff
@@ -118,12 +118,21 @@ def parse_macros(*args)
...
           end
         end
       end
+
+      def add_macro_from_hash(macros, spec)
+        spec.each do |k, v|
+          macros << [
+            k,
+            v.is_a?(Array) ? v : [v]
+          ]
```
Comment (solnic) [inline lib/dry/validation/rule.rb]: could you inline this?

**Comment by waiting-for-dev** [comment]
Do you mean the whole `each` block or just the array push? In the first
case, maybe line length is too high...

**Comment by solnic** [comment]
@waiting-for-dev just the array push. re line-length, I think these days with gigantic screen resolutions `100` is a sensible value, but we can change it to more standard 80

**Comment by waiting-for-dev** [comment]
Ok @solnic , I forced a push with the change style. About line length, I'm always nostalgic of the old times, so I like the 80 limit, even if I recognize it's sometimes insane :smile:

**Pull request merged**